modules/async_worker.py

```diff
@@ -426,6 +426,10 @@ def handler(async_task):
             extra_negative_prompts = negative_prompts[1:] if len(negative_prompts) > 1 else []
 
             progressbar(async_task, 3, 'Loading models ...')
+
+            # Parse lora references from prompt
+            loras = parse_lora_references_from_prompt(prompt, loras, loras_limit=modules.config.default_max_lora_number)
```

Comment:
```suggestion
            loras = parse_lora_references_from_prompt(prompt, loras, modules.config.default_max_lora_number)
```

+ don't you agree we should also replace the lora notation in the prompt for correct metadata handling, logging etc.?